### PR TITLE
Fixed bug that occurred when default 'python' refers to python3

### PR DESCRIPTION
**Pull request by aaxu**
Changed the she-bang so that it should now refer to python 2 (since we don't have python3 support as of now)

### CODE REVIEW

File: ci.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
```
Comment (dschuyler) [inline ci.py]: On Mac OSX this generates:
/usr/bin/python2: bad interpreter: No such file or directory
(sadly)

Comment (aaxu) [inline ci.py]: Hmm how does python get referenced in mac then? I don't own a mac so I can't implement that part well. At best, I can maybe detect the OS type and use python if it's OS and python2 if Linux.

**Comment by dschuyler** [comment]
python2.7 does exist on OSX. Is that available on other platforms?

**Comment by dschuyler** [comment]
Maybe the install.sh could set up the #! line after determining which command to run?

Or the execution could start with a shell script rather than a python script. (essentially doing that executable determination on every startup rather than at install time -- this would only be ok if it were very fast to do).

**Comment by aaxu** [comment]
Ohh yeah on both of my systems, I have both python2.7 and python2 which are two different files (/usr/bin/python2.7 and /usr/bin/python2). They both start up the same python version for me. I'm not sure if this is the case for all the systems, but if that works, then this could be a solution.

**Pull request merged**